addons/sourcemod/translations/superzombiefortress.phrases.txt

```diff
@@ -387,6 +387,11 @@
 	{
 		"en"		"{orange}The Killing Gloves of Boxing {red}gives 5 seconds of minicrits on kill."
 	}
+
+	"Melee_GlovesofRunningUrgently"
+	{
+		"en"		"{orange}The Gloves of Running Urgently {green}has health drain replaced with Marked-For-Death while active."
```

Comment:
Even if everyone sees the whole as a direct upgrade, there still a downside to colour the text red.
`{orange}The Gloves of Running Urgently {green}has its health drain disabled. {red}Marked-For-Death while active.`